dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScope.kt

```diff
@@ -65,16 +76,46 @@ internal class RumSessionScope(
 
     // region Internal
 
+    private fun onApplicationDisplayed(
+        writer: Writer<RumEvent>
+    ) {
+        applicationDisplayed = true
+        val now = System.nanoTime()
+        val startupTime = Datadog.startupTimeNs
+        val context = getRumContext()
+        val actionEvent = ActionEvent(
+            date = applicationStart,
+            action = ActionEvent.Action(
+                type = ActionEvent.Type1.APPLICATION_START,
+                id = sessionId,
+                loadingTime = max(now - startupTime, 1L)
+            ),
+            view = ActionEvent.View(id = RumContext.NULL_SESSION_ID, url = ""),
```

Comment:
I would rather send this as null if it is possible...

Reply:
It's not possible, the schema requires a view.